docarray/documents/audio.py

```diff
@@ -94,11 +96,26 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[AudioUrl]
-    tensor: Optional[AudioTensor]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[AudioBytes]
-    frame_rate: Optional[int]
+    url: Optional[AudioUrl] = Field(
+        description='''The url to a (potentially remote) audio file that can be
+        loaded''',
+        example='https://github.com/docarray/docarray/blob/main/tests/toydata/hello.mp3?raw=true',
+    )
+    tensor: Optional[AudioTensor] = Field(
+        description='''Tensor object of the audio which be specified to one of
```

Comment:
is this gramatically correct to say which be specified?

Reply:
apologies for the typo, should be `which can be specified to one of`